config/config.schema.json

```diff
@@ -1568,23 +1574,30 @@
           "type": "boolean"
         },
         "azureAdInstance": {
-          "type": "string"
+          "type": "string",
+          "description": "The Azue Ad Instance that holds the ARM Application (see armAppId property)"
         },
         "firstPartyAppId": {
-          "type": "string"
+          "type": "string",
+          "description": "This is the app ID that is used in MISE audit logs. Not used for verification of any kind."
         },
         "armAppId": {
           "type": "string",
           "description": "The valid app ID for mise, which is the ARM Application Id, this is common across all clouds (MSIT/AME/FF)"
         },
+        "tenantId": {
+          "type": "string",
+          "description": "The tenant ID where the ARM Application (see armAppId property) lives. Usually AME"
+        },
+        "bearerTokenAuthorityFQDN": {
+          "type": "string",
+          "description": "The FQDN of the bearer token authority"
+        },
```

Comment:
nit: Now that we're supporting multiple configurations of MISE and it's established kinds of trusts - is it worth putting them into buckets? e.g. a bucket for the ARM PoP tokens, and another for Geneva Actions? The documentation goes a long way here to be sure - but I wonder if we could make this more self-documenting that we've got exactly two kinds of config we care about?